bin/ovs-offline

```diff
@@ -324,21 +327,37 @@ do_collect-sos-ovs() {
 
     save_dir=${WORKDIR}/restore_flows
     mkdir -p $save_dir
+    
+    if [[ -n ${OVS_RUNDIR} && -f $imported_db_file ]]; then
+        # The DB already exists, we don't have to check the sosreport for the bridges/tlv-map/groups, etc. Some older versions of `sos` do not collect this.
+        # We do still need the flows from the sosreport, which should be present in most recent versions of sos (tested >= 3.8).
+        # If OVS_RUNDIR is present in the environment variables, then it means we have sourced the commands, so we can use `ovs-*` commands directly in the shell.
+        bridges=$(ovs-vsctl -t 5 list-br)
+        for br in $bridges; do
+            # Get the highest level of supported protocol for each bridge.
+            local high_ver=$(ovs-vsctl get Bridge $br protocols | tr -d '[|]' | awk '{print $NF}')
+            ovs-ofctl -O $high_ver dump-tlv-map $br > $save_dir/ovs-ofctl_-O_${high_ver}_dump-tlv-map_${br} || error "Error in executing the ovs-ofctl dump-tlv-map command. Please check your DB file."
```

Comment:
If the DB was imported but the sos report has not yet been parsed, there is no way this command will return anything, right?
`ovs-ofctl` is run against ovs-vswitchd, not the DB.